tests/main/views/test_search.py

```diff
@@ -443,8 +450,11 @@ def test_search_results_show_aggregations_by_parent_category(self):
             assert expected_lot_counts[category_name] == int(number_of_services)
 
     def test_search_results_sends_aggregation_request_without_page_filter(self):
-        data_api_client.find_frameworks.return_value = {'frameworks': [self._get_framework_fixture_data('g-cloud-9')
-                                                                       ['frameworks']]}
+        self.data_api_client.find_frameworks.return_value = {
+            'frameworks': [
+                self._get_framework_fixture_data('g-cloud-9')['frameworks']
```

Comment:
Not sure there's much value in this having its own line. Could be 
```
self.data_api_client.find_frameworks.return_value = {
            'frameworks': [self._get_framework_fixture_data('g-cloud-9')['frameworks']]
        }
```

Reply:
Yeah... I only did it because I dislike having the `]]` on the end 😸 Happy to change.